Docs/Getting Started Guide.md

```diff
@@ -320,24 +311,36 @@ func fetchGitHubRepositories(state: State, store: Store<State>) -> Action? {
 }
 ```
 
-In the example above, we're using an `enum` to represent the different states of a single state slice that depends on a network request (e.g. loading, result available, network request failed). There are many different ways to model states of a network request but it will mostly involve using multiple dispatched actions at different stages of your network requests.
+In the example above we're using an `enum` to represent the different states of a single state slice that depends on a network request. It may look something like this:
+
+```swift
+enum GitHubFetchRequestState {
+    case Loading
```

Comment:
To comply with Swift naming conventions, I suggest lowercasing the cases to `case loading`, `case repositories`, `case networkError`.